source/databases-collections.txt

```diff
@@ -99,14 +96,14 @@ designating its type. Index key types include:
 
 .. note::
 
-   You can open a :ref:`MongoDB Playground <vsce-crud>` 
-   pre-configured to create an index by hovering over the 
-   :guilabel:`Indexes` label in the navigation panel and clicking the 
-   :icon-fa4:`plus` icon that appears.
+   You can open a :ref:`MongoDB Playground <vsce-crud>` pre-configured
+   to create an index by hovering over the :guilabel:`Indexes` label
+   in the navigation panel and clicking the :icon-fa4:`plus` icon that
+   appears.
 
 .. seealso::
 
-   For more information about MongoDB indexes, see the documentation in
+   To learn more about MongoDB indexes, see the documentation in
```

Comment:
[suggestion] do we need to say "the documentation in the {abc}"? Can we just say: To learn more, see <link>?